datacommons_pandas/examples/df_builder.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Basic examples for building pandas objects using the Data Commons Pandas API."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import datacommons_pandas as dcpd
+
+
+def main():
+
+    print("""
```

Comment:
still would be nice to parameterize the examples

Reply:
I feel like it's not worth it when there's only 2 each for the 3 functions?

Reply:
it is easier to add more examples in the future and do changes too